src/AspNetCore/nativeaot.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+cd Example
+dotnet restore
+dotnet publish -c Release -r linux-x64 -p:PublishNativeAot=True
+# Uncomment to get rid of debugging symbols in executable
```

Comment:
Is there real situation when someone wants to keep this information for release build?

Reply:
I do not know if `strip` is part of most Linux distros, so I prefer that developer read script and find that it should run this command

Reply:
I would change script behavior from opt-in to opt-out and add comment regarding availability of this command.